lib/usd/hdMaya/CMakeLists.txt

```diff
@@ -53,8 +31,9 @@ target_compile_definitions(${TARGET_NAME}
 # -----------------------------------------------------------------------------
 target_include_directories(${TARGET_NAME} 
     PRIVATE
-        ${CMAKE_BINARY_DIR}/include
         ${CMAKE_CURRENT_SOURCE_DIR}
+    INTERFACE
+        ${CMAKE_BINARY_DIR}/include
```

Comment:
Why is this required? Shouldn't this be inherited, since it was declared a PUBLIC include dir for the project?

Reply:
@elrond79 Correct, hdMaya automatically gets this when it links against mayaUsd.  
Sorry I must have missed to remove it from here. Will fix it.

I am also preparing some notes for your above comment. Will reply sometimes today.

Reply:
see 1473f12e